pkgs/by-name/de/deezer-enhanced/package.nix

```
@@ -0,0 +1,125 @@
+{
+  ### Tools
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  makeWrapper,
+  autoPatchelfHook,
+  dpkg,
+  gnutar,
+
+  ### Libs
+  xorg,
+  libxkbcommon,
+  glib,
+  nss,
+  dbus,
+  at-spi2-atk,
+  cups,
+  gtk3,
+  pango,
+  cairo,
+  expat,
+  systemdLibs,
+  alsa-lib,
+  nwjs,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "deezer-enhanced";
+  version = "1.2.0";
```

Is there something wrong with v1.3.0? Why not use the latest version?

```suggestion
  version = "1.3.0";
```

---

I had put it under this version because I had errors concerning libGL.so during my tests

---

I have just added it and changed the version